app/templates/settings/general.hbs

```diff
@@ -39,13 +39,30 @@
 
                 {{#if showUploadLogoModal}}
                     {{gh-fullscreen-modal "upload-image"
-                            model=(hash model=model imageProperty="logo")
+                            model=(hash model=model imageProperty="logo" accept=imgMimeTypes extensions=imgExtensions allowUrlInput=true)
```

Comment:
Taking a deeper look at this I think it's because inside `gh-fullscreen-modal` the attributes are always passed on to the `gh-image-uploader` component so if nothing is passed in to the modal component the defaults on the uploader component will be overridden with `undefined`.

One way around it would be to use the `didReceiveAttrs` hook in the `gh-image-uploader` to check if the provided attributes are null or undefined then set them to the defaults, e.g.:

```
accept: null,
extensions: null,

_defaultAccept: 'image/gif,image/jpg,image/jpeg,image/png,image/svg+xml',
_defaultExtensions: ['gif', 'jpg', 'jpeg', 'png', 'svg'],

didReceiveAttrs() {
    if (!this.get('accept') { this.set('accept', this.get('_defaultAccept')); }
    if (!this.get('extensions') { this.set('extensions', this.get('_defaultExtensions')); }
}
```